.github/workflows/build_base.yml

```
@@ -143,25 +143,312 @@ jobs:
+    needs: [ upload-image, get-date-release-name, modify-rpi-image ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Update base-image-url in build_img_en.yml
+        run: |
+          sed -i '/^[^#]*base-image-url:/ {/sed/! s|base-image-url: .*|base-image-url: https://github.com/OpenVoiceOS/raspOVOS/releases/download/${{ needs.get-date-release-name.outputs.release_name }}/${{ needs.modify-rpi-image.outputs.image-path }}|}' .github/workflows/build_img_en.yml
+
+      - name: Check for changes
+        id: check-changes
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:29: Double quote to prevent globbing and word splitting

(shellcheck)